src/plugins/relay/relay-config.c

```diff
@@ -816,6 +817,11 @@ relay_config_init ()
         "backlog_since_last_disconnect", "boolean",
         N_("display backlog starting from last client disconnect"),
         NULL, 0, 0, "on", NULL, 0, NULL, NULL, NULL, NULL, NULL, NULL);
+    relay_config_irc_backlog_since_last_message = weechat_config_new_option (
+        relay_config_file, ptr_section,
+        "backlog_since_last_disconnect", "boolean",
```

Comment:
I just noticed, this line should be 
        <code>"backlog_since_last_message", "boolean",<\code>